frontend/app/src/store/tasks/index.ts

```diff
@@ -184,6 +185,14 @@ export const useTaskStore = defineStore('tasks', () => {
     }
   };
 
+  const cancelTaskByTaskType = async (taskTypes: TaskType | TaskType[]): Promise<void> => {
```

Comment:
I wonder if we could do the change in a way that we can use abort signals in the request chain

Reply:
for the historic price it would be more complicated, since we have many layer of request (the cache)